tests/factories/contacts/cht/district-hospital.js

```diff
@@ -0,0 +1,12 @@
+const Factory = require('rosie').Factory;
+
+Factory.define('cht_district_hospital')
+  .attr('_id', 'dist1')
+  .attr('type', 'district_hospital')
+  .attr('is_name_generated', 'true')
+  .attr('name', 'District Hospital 1')
+  .attr('external_id', 'external id 1')
+  .attr('notes', 'This is a note')
+  .attr('place_id', '04789')
+  .attr('reported_date', Date.now());
```

Comment:
Same comments here about the `_id`, other fixed values, and the `Date.now()`. I'll skip adding those same comments here and leave it to you to take the points from the earlier comments and apply them universally.